### ISSUE TITLE
scrollbar

### ISSUE THREAD

**Issue by yo8z6gv** [kotatogram/kotatogram-desktop#208]
можете добавить в настройках кастомизацию скроллбара?
мне нужно чтобы по ширине в чате было 18 пикселей, вместо 12 пикселей
чтобы были стрелки сверху, снизу для пролистывания 
поддержка в чате СКМ для пролистывания, как в браузере. хроме лучше всего Средняя Кнопка Мыши сделана.
более контрастнее ползунок

когда слишком много прогружено в чате смещение на пиксель иногда пролистывает больше чем умещается, особенно актуально это в маленьком окне. 
ну и возможно кнопку "обновить чат")) чтобы остаться на текущем сообщении в чате, но выгрузить сверху/снизу сообщения, обновить позиционирования отображения с фактическим размещением (если отправлять файлы до 1 мб, то последовательность отправки никогда не соответствует фактическому размещению, в итоге файлы все время отправляются в неправильном порядке; сейчас нужно перезапускать программу после таких отправок чтобы посмотреть как выглядят отправленные файлы; или когда пересылаешь 100 сообщений, в чате ты видишь их снизу вверх, когда на самом деле они сверху вниз)

ps у меня программируемая мышь и на ней кнопки отвечают за функции режимов прокрутки колеса мыши вверх/вниз (1 порог - 1 нажатие, x? порогов 1 нажатие, удержание постоянная прокрутка с порогом и промежутком и т.д..) кароче, я обычно не пользуюсь колесом мыши когда можно через скм прокручивать, а когда слишком много прокручиваю, то удобнее использовать home/page up; end/page down. home/end в телеграме не работают, а page up/page down прокручивают выравнивают по пикселям, в итоге иногда буквально текст разделен пополам)) и приходится все таки чуть смещать через скролл бар или стрелочки

**Comment by ilya-fedin** [comment]
> можете добавить в настройках кастомизацию скроллбара?

Ну, если ты напишешь код для этого и сделаешь PR - без вопросов.

**Comment by antshar** [comment]
Согласен, было бы очень удобная вещь, особенно для создания длинных скриншотов.
